symbolic-debuginfo/src/sourcebundle.rs

```diff
@@ -1185,13 +1191,20 @@ where
         S: AsRef<str>,
         R: Read,
     {
+        let mut buf = String::new();
+
+        if let Err(e) = file.read_to_string(&mut buf) {
```

Comment:
not too happy about allocating/copying another string here, but meh I guess its not too bad